frontend/src/views/ProjectView.vue

```diff
@@ -192,13 +237,18 @@ async function dropFilesHandler(fileList: FileList, uploadFileSize: number) {
     return
   }
 
+  if (uploadFileSize == 0) {
+    storeApp.alertsWarning("Uploading file(s)'s size is zero")
```

Comment:
Would also be good to be able to know if one file is a folder ; in order to provide a better error message (`Network Error` at the moment).

Reply:
Determining if an upload item is a folder is quite tricky. Every upload item has a property called `type` and this property is just a MIME type. For the folder and anything that doesn't have a MIME type, this property is empty.

I think we don't want users to upload something that doesn't have the mime type, so we can determine if the `type` is empty and alert a message like "The XXXX/Files are not supported."

Reply:
Let's do this until we have user complain about _valid_ files raising this. We can use a folder-oriented wording like `Folders cannot be uploaded directly`. It will be easier to assist users